docs/en/Configuration.md

```diff
@@ -75,6 +75,20 @@ use SilverStripe\CMS\Model\SiteTree;
   // Get the setting
   $loginRequired = singleton(SiteTree::class)->getCommentsOption('require_login');
 ```
+### spam control
+
+you can create a comment.yml file with the following content:
+
+```yml
+---
+Name: 'app_comments_spam'
+---
+
+Page:
+  extensions:
+    - SilverStripe\SpamProtection\Extension\CommentSpamProtection
```

Comment:
That extension seems to come from a separate module - please point developers to the relevant module instead, and to any documentation that module has for its usage.